src/index.js

```diff
@@ -24,8 +24,12 @@ const checkResponse = async response => {
     if (response.ok && isOK) return response.json();
     throw await response.json();
   }
-  if (response.ok && isOK) return response.text();
-  throw await response.text();
+  if (response.ok && isOK) {
+    let responseText = await response.text();
+    let authFailed = responseText.includes('Authorization Error');
```

Comment:
String.prototype.includes might not always be polyfilled, so IE 11 will fail on this line. Please switch to String.prototype.indexOf.